packages/lib/src/validations/VanillaContracts.ts

```diff
@@ -0,0 +1,19 @@
+import Contract from '../artifacts/Contract';
+import ContractAST from '../utils/ContractAST';
+
+let VANILLA_CONTRACTS = '@openzeppelin/contracts/';
+
+export function importsVanillaContracts(contract: Contract, buildArtifacts?: any): string[] | null {
+  const ast = new ContractAST(contract, buildArtifacts, { nodesFilter: ['ContractDefinition'] });
+  const illegalImports = [...ast.getImports()]
+    .filter(i => i.startsWith(VANILLA_CONTRACTS))
+    .map(i => i.slice(VANILLA_CONTRACTS.length))
+    .map(i => i.replace(/^contracts\//, ''));
+
+  return illegalImports.length > 0 ? illegalImports : null;
```

Comment:
Why `null` here? (I feel that we've had this conversation before already. 🤔) We should favor `undefined` because it's what's used by TypeScript for optional properties and arguments. This isn't affecting us currently because we're not using TypeScript in strict mode, but we should and will. For example, in this same PR you defined an optional property `importsVanillaContracts?: string[]`, to which you're assigning the result of this function. This wouldn't type in strict mode because of `null`.

Reply:
> Why null here? (I feel that we've had this conversation before already. thinking)

Because I forgot about the conversation we had already. Thanks for pointing it out, I'll do my best to break the force of habit.